utils/install.py

```diff
@@ -1569,13 +1566,34 @@ def main(args):
     args.path = abspath(args.path)
 
     if args.tf_version is None:
-        args.tf_version = args.version
+        if VersionString(args.version).compare("0.12.0") == -1:
+            args.tf_version = "TF-2.6.0"
+        elif VersionString(args.version).compare("0.13.0") == -1:
+            args.tf_version = "TF-2.6.0-CC"
+        elif VersionString(args.version).compare("0.13.0") >= 0:
+            args.tf_version = "TF-2.12.0-CC"
+        else:
+            reraise("Should not reach here")
 
     if args.tf_deps_version is None:
-        args.tf_deps_version = args.version
+        if VersionString(args.version).compare("0.12.0") == -1:
+            args.tf_deps_version = "TF-2.6.0"
+        elif VersionString(args.version).compare("0.13.0") == -1:
+            args.tf_deps_version = "TF-2.6.0-CC"
+        elif VersionString(args.version).compare("0.13.0") >= 0:
+            args.tf_deps_version = "TF-2.12.0-CC"
```

Comment:
Maybe `#DNE` here?

Reply:
There won't be any reason for `does not exist` as we have covered all the cases I guess.